client/components/domains/register-domain-step/index.jsx

```diff
@@ -265,10 +265,9 @@ var RegisterDomainStep = React.createClass( {
 					} );
 				},
 				callback => {
-					if ( abtest( 'domainSearchResultsCount' ) === 'moreResults' ) {
-						SUGGESTION_QUANTITY = 10;
-					}
-
+					
+					SUGGESTION_QUANTITY = 10;
```

Comment:
This needs to be moved to https://github.com/Automattic/wp-calypso/blob/444c841ed299eab9e926af6a66c93d3a26051fa6/client/components/domains/register-domain-step/index.jsx#L33